cmd/traffic/cmd/manager/managerutil/envconfig.go

```diff
@@ -60,6 +58,9 @@ type Env struct {
 	AgentEnvoyLogLevel       string                     `env:"AGENT_ENVOY_LOG_LEVEL,    parser=logLevel,       defaultFrom=AgentLogLevel"`
 	AgentEnvoyServerPort     uint16                     `env:"AGENT_ENVOY_SERVER_PORT,  parser=port-number"`
 	AgentEnvoyAdminPort      uint16                     `env:"AGENT_ENVOY_ADMIN_PORT,   parser=port-number"`
+	AgentRunAsUser           *int64                     `env:"AGENT_RUN_AS_USER,        parser=maybe-int64,    default="`
+	AgentRunAsGroup          *int64                     `env:"AGENT_RUN_AS_GROUP,       parser=maybe-int64,    default="`
+	AgentRunAsNonRoot        *bool                      `env:"AGENT_RUN_AS_NON_ROOT,    parser=maybe-bool,     default="`
```

Comment:
Although UID and GID seems to be defined as `uint` in Linux, Kubernetes uses pointers too.
I think this is because an user might decide to not set an ID, and `-1` might have different effects:
https://man7.org/linux/man-pages/man2/setreuid.2.html

Obviously leaving it as `0` when the user decides not to specify that is also an "error", as this will run the container as root